hackathon_site/dashboard/frontend/src/components/acknowledgement/AcknowledgementForm/Waver.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import Typography from "@material-ui/core/Typography/Typography";
+import { hackathonName } from "constants.js";
+
+const Waver = (
```

Comment:
Consider using MUI dialog? then we will have the transition animation without using react-transition-group

Reply:
the waiver uses teh mui modal component and isn't attached to any transition group, the acknowledgement form is part of the transition though